src/app/ui/NewTaskForm.tsx

```diff
@@ -261,14 +261,14 @@ export const NewTaskForm = ({
                 <Typography
                   variant="bodySm"
                   sx={{
-                    color: (theme) => theme.color.gray[600],
+                    color: (theme) => (tempAssignee ? theme.color.gray[600] : theme.color.text.textGray),
```

Comment:
@arpandhakal should't both text color come from `theme.color.text`? looks bit inconsistent

cc: @aatbip

Reply:
We have included all the colors in the `theme.color` while writing the theme config initially. So I think it is better if we can add `textGray` in the `theme.color.gray` instead of `theme.color.text`.

Reply:
added a new color in text.color.gray : 550 , which technically lies between 500 and 600. applied requested changes please re-review @saugatmaharjan @aatbip